### PR TITLE
Feature/n UI 6084 add e2e to wizardv2

### PR DESCRIPTION
**Pull request by DmytroPutrin**

### CODE REVIEW

File: packages/bits/spec/components/wizard-v2/wizard-v2.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+            expect(await wizard.isPresent()).toEqual(false);
+        });
+    });
+
+    describe("restore state", () => {
+        it("should open dialog on last step", async () => {
+            await openWizardDialogBtn.click();
+
+            const wizard = Atom.find(WizardV2Atom, "nui-wizard-v2-horizontal-dialog");
```
Comment (dmytrofirsov) [inline packages/bits/spec/components/wizard-v2/wizard-v2.e2e.ts:96]: You don't have to find the atom every time at the start of each test. YOu can have it found once at the `beforeAll` hook at the very top.

File: packages/bits/spec/components/wizard-v2/wizard-v2-footer.atom.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    public getFinishButton(): Promise<ButtonAtom> {
+        return this.getButtonAtom(".nui-button.complete");
+    }
+
+    private getButtonAtom(query: string): Promise<ButtonAtom> {
+        const button = this.root.element(by.css(query));
+
+        return Promise.resolve(new ButtonAtom(button));
```
Comment (dmytrofirsov) [inline packages/bits/spec/components/wizard-v2/wizard-v2-footer.atom.ts:30]: Can't we simply return the new Button atom here?

File: packages/bits/spec/components/wizard-v2/wizard-v2.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        await Helpers.prepareBrowser("wizard-v2/wizard-test");
+        await Helpers.disableCSSAnimations(Animations.ALL);
+
+        wizard = Atom.find(WizardV2Atom, "nui-wizard-v2-horizontal");
+        openWizardDialogBtn = Atom.find(ButtonAtom, "nui-wizard-dialog-trigger");
+    });
+
+    describe("by default",  () => {
+        it("should wizard be preset", async () => {
+            expect(await wizard.isPresent()).toBeTruthy(true);
```
Comment (dmytrofirsov) [inline packages/bits/spec/components/wizard-v2/wizard-v2.e2e.ts:22]: If this condition is verified with the unit tests it is probably doesn't have to be here

**Review by dmytrognatyshyn — APPROVED** [review]

File: packages/bits/spec/components/wizard-v2/wizard-v2.visual.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import { Atom } from "../../atom";
+
+import { Helpers } from "../../helpers";
+import { Camera } from "../../virtual-camera/Camera";
+import { ButtonAtom } from "../button/button.atom";
+import { WizardV2Atom } from "./wizard-v2.atom";
+
+const name: string = "Wizard V2";
+
+fdescribe(`Visual tests: ${name}`, () => {
```
Comment (Vita-Karas) [inline packages/bits/spec/components/wizard-v2/wizard-v2.visual.ts:11]: fdescribe

**Review by Vita-Karas — APPROVED** [review]

**Review by ivankozlovskyi — APPROVED** [review]

**Pull request merged**